LICENSE

```diff
@@ -1,6 +1,10 @@
-MIT License
+Copyright (c) IBM Corp. 2012,2017. All Rights Reserved.
```

Comment:
should this be 2012,2018? Same thing for the swagger connector

Reply:
should be. thanks.